src/components/search-bar/search-bar.tsx

```diff
@@ -5,27 +5,37 @@ import { useText } from "../../core/utils/text";
 
 export interface SearchBarProps {
   getInputProps: UseComboboxPropGetters<unknown>["getInputProps"];
+  getLabelProps: UseComboboxPropGetters<unknown>["getLabelProps"];
 }
 
-const SearchBar: React.FC<SearchBarProps> = ({ getInputProps }) => {
+const SearchBar: React.FC<SearchBarProps> = ({
+  getInputProps,
+  getLabelProps
+}) => {
   const t = useText();
   return (
     <>
-      {/* The downshift combobox uses prop spreading by design */}
+      {/* The downshift combobox uses prop spreading by design & associated control is desctructured too */}
+      {/* eslint-disable-next-line jsx-a11y/label-has-associated-control, react/jsx-props-no-spreading */}
```

Comment:
jsx-a11y/label-has-associated-control <- think you can avoid that by wrapping the label around the input <label><input></l...

Reply:
Mmmm @sinejespersen it doesn't seem like it. Check out the screenshot where I wrap it around the input but still get the error.
![image](https://user-images.githubusercontent.com/28546954/206397862-3e7aadbc-f39f-4e68-acb4-f8393ee2fee2.png)

Reply:
oh sorry, add the htmlFor instead... I think i've made it work with some sort of combination of the two... https://developer.mozilla.org/en-US/docs/Web/API/HTMLLabelElement/htmlFor

Reply:
@sinejespersen the thing is that Downshift (the library we use for the autosuggest + search input field) gives the input the htmlFor attribute on runtime (see the `{...getLabelProps()}`). Eslint is not happy with it because it cannot see the attribute hardcoded (which is understandable), but we decided (there was indeed an internal discussion about this) to utilize the library instead of hard coding it. Thus the eslint-disable.

Reply:
ah I see